src/model/OWL.plt

```diff
@@ -0,0 +1,172 @@
+  assert_false(is_data_property(test:'p')),
+  assert_false(is_data_property(test:'bIndiv')),
+  assert_true(is_data_property(test:'P')).
+
+test('is_functional_property') :-
+  assert_true(is_functional_property(test:'p')),
+  assert_false(is_functional_property(test:'s')),
+  assert_false(is_functional_property(test:'r')),
+  assert_false(is_functional_property(test:'DoesNotExists')).
+
+test('is_transitive_property') :-
+  assert_false(is_transitive_property(test:'p')),
+  assert_false(is_transitive_property(test:'s')),
+  assert_true(is_transitive_property(test:'r')),
+  assert_false(is_transitive_property(test:'DoesNotExists')).
+
+test('is_symmetric_property') :-
+  assert_false(is_symmetric_property(test:'p')),
+  assert_true(is_symmetric_property(test:'s')),
+  assert_false(is_symmetric_property(test:'r')),
+  assert_false(is_symmetric_property(test:'DoesNotExists')).
+
+test('has_inverse_property') :-
+  assert_true(has_inverse_property(test:'r',test:'rInv')),
+  assert_true(has_inverse_property(test:'rInv2',test:'r')),
+  assert_false(has_inverse_property(test:'p',test:'r')).
+
+test('has_property_chain') :-
+  assert_true(has_property_chain(test:'t',[test:'t1',test:'t2',test:'t3'])),
+  assert_false(has_property_chain(test:'t',[test:'t1',test:'t2'])),
+  assert_false(has_property_chain(test:'t',[test:'p',test:'r'])),
+  assert_false(has_property_chain(test:'p',[test:'t1',test:'t2',test:'t3'])).
+
+test('has_disjoint_class1') :-
+  assert_true(has_disjoint_class(test:'DisjClsChain1',test:'DisjClsChain2')),
+  assert_true(has_disjoint_class(test:'DisjClsChain2',test:'DisjClsChain3')),
+  assert_false(has_disjoint_class(test:'DisjClsChain1',test:'DisjClsChain3')),
+  assert_false(has_disjoint_class(test:'A',test:'B')),
+  assert_true(has_disjoint_class(test:'DisjClsChain1Sub',test:'DisjClsChain2Sub')).
+
+% Test OWL2 AllDisjointClasses
+test('has_disjoint_class2') :-
+  assert_true(has_disjoint_class(test:'DisjCls1',test:'DisjCls2')),
+  assert_true(has_disjoint_class(test:'DisjCls3',test:'DisjCls2')),
+  assert_true(has_disjoint_class(test:'DisjCls1',test:'DisjCls3')),
+  assert_true(has_disjoint_class(test:'DisjCls1Sub',test:'DisjCls2Sub')).
+
+test('has_equivalent_class') :-
+  assert_true(has_equivalent_class(test:'ASub',test:'ASubEq')),
+  assert_true(has_equivalent_class(test:'ASubEq',test:'ASub')),
+  assert_true(has_equivalent_class(test:'EqClsChain1',test:'EqClsChain2')),
+  assert_true(has_equivalent_class(test:'EqClsChain2',test:'EqClsChain3')),
+  assert_true(has_equivalent_class(test:'EqClsChain1',test:'EqClsChain3')),
+  assert_false(has_equivalent_class(test:'A',test:'B')).
+
+test('same_as') :-
+  assert_true(same_as(test:'bIndiv',test:'bIndiv')),
+  assert_true(same_as(test:'aIndiv',test:'bIndiv')),
+  assert_true(same_as(test:'aIndiv',test:'aIndiv')),
+  assert_false(same_as(test:'aIndiv',test:'indiv')),
+  assert_false(same_as(test:'aIndiv',test:'doesNotExists')).
+
+:- end_tests(model_OWL).
```

Comment:
newline missing

Reply:
You mean adding a newline at the end of the file? Why? Is it the style that is generally used in Knowrob?

Reply:
github complains.
not really needed but the github UI does not like it :P